samples/azure/storage-persistence/ASP_2/Client/Client.csproj

```diff
@@ -8,11 +8,11 @@
     <ProjectReference Include="..\Shared\Shared.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="NServiceBus.Azure.Transports.WindowsAzureStorageQueues" Version="8.*" />
+    <PackageReference Include="NServiceBus.Transport.AzureStorageQueues" Version="9.*" />
```

Comment:
We killed the sample on the ASP update branch and also moved then

Reply:
OK, I'll remove the sample from this PR then.